src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/LegalRepresentativeUpdateDetailsConfirmationTest.java

```diff
@@ -0,0 +1,90 @@
+package uk.gov.hmcts.reform.iacaseapi.domain.handlers.postsubmit;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.hamcrest.Matchers.containsString;
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.reset;
+import static org.mockito.Mockito.when;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.AsylumCase;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.Event;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.callback.Callback;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.callback.PostSubmitCallbackResponse;
+
+@RunWith(MockitoJUnitRunner.class)
+@SuppressWarnings("unchecked")
```

Comment:
do you really need this suppression?

Reply:
The `reset(callback);` on line 75 will raise a warning `Unchecked generics array creation for varargs parameter`, the reason `@SuppressWarnings(...)` is used is because I am unable to use `@SafeVarargs` on a method with a `@Test` annotation.
This is just to make sonarcloud happy and reset the callback to original form without mocks.